client/src/app/views/admin/views/admin-profile/admin-profile-logs/admin-profile-logs.component.html

```diff
@@ -1,26 +1,35 @@
-<table class="table">
+<table class="table" style="table-layout: fixed; width: 100%;">
```

Comment:
Avoid styling through style attribute extract to a class

Reply:
Removed; seems like it isn't needed anyways.